src/main/java/com/group12/ciserver/controller/EventController.java

```diff
@@ -16,4 +24,37 @@
         log.info("Received push event, pushEvent={}", pushEvent);
         return ResponseEntity.noContent().build();
     }
+
+    @GetMapping("/history")
+    @ResponseBody
+    public ArrayList<BuildInfo> buildHistory() {
+
+        DatabaseWrapper db = new DatabaseWrapper();
```

Comment:
Instead of creating a new `DatabaseWrapper` object each time the endpoint is called, you can Autowire it in the field of this class in order to reuse the same instance.

Do something like:
`
@Autowire
private DatabaseWrapper databaseWrapper;
`
Spring will inject the DatabaseWrapper to this class. Don't forget to annotate DatabaseWrapper with `@Component` on class level.